examples/dask/community_demo/demo_day_notebook.ipynb

```diff
@@ -1,18 +1,17 @@
 {
  "cells": [
   {
+   "metadata": {},
    "cell_type": "code",
-   "execution_count": 1,
-   "metadata": {
-    "ExecuteTime": {
-     "end_time": "2023-05-22T22:27:16.952106Z",
-     "start_time": "2023-05-22T22:27:15.116241Z"
-    },
-    "pycharm": {
-     "name": "#%%\n"
-    }
-   },
    "outputs": [],
+   "execution_count": null,
+   "source": "#!pip install pandas \"sf-hamilton[dask,visualization]\""
```

Comment:
Do we need to escape `\"` quotation marks? Also, might no need the comment `#`

Reply:
this is notebook escaping because this is JSON.